stories/qa/card/card.stories.html

```diff
@@ -59,7 +59,7 @@ <h2 class="card-title">Title</h2>
 <!-- States -->
 <section class="contentSection">
 	<h2 class="card-title">States</h2>
-	<a class="card mod-clickable">
+	<a href class="card mod-clickable">
```

Comment:
We need to rethink this to avoid the wrap-around link.

Reply:
Agreed, but it's just a fix on a deprecated mod. I guess we are going to improve `mod-action` in the future.